jetty-server/src/main/java/org/eclipse/jetty/server/session/AbstractSessionDataStoreFactory.java

```diff
@@ -19,8 +19,8 @@
 public abstract class AbstractSessionDataStoreFactory implements SessionDataStoreFactory
 {
 
-    int _gracePeriodSec;
-    int _savePeriodSec;
+    int _gracePeriodSec = AbstractSessionDataStore.DEFAULT_GRACE_PERIOD_SEC;
+    int _savePeriodSec = AbstractSessionDataStore.DEFAULT_SAVE_PERIOD_SEC;
```

Comment:
These are embedded mode changes.
It used to be 0 for each.

Reply:
See above